telosnetwork/telos-wallet
# #285 | Currency input on send page

# Fixes #285
## Description
PR adds the currency input to send page. it also updates balances logic to prevent getting price from coingecko in certain cases, such as during gas estimation

Note: sending WTLOS appears to be broken in staging, which means it is also broken in this PR
Note: it may be helpful to hide whitespace changes when reviewing this: https://github.com/telosnetwork/telos-wallet/pull/338/files?diff=split&w=1

## Test scenarios
**Testnet - only have price for TLOS**
- go to https://deploy-preview-338--wallet-staging.netlify.app/
- log in
- click send page button
- try sending different amounts of tokens to yourself or another account
    - the amount to send should match what you typed in
- make sure you also test with the currencies swapped (when you type in a USD value)
- send button should be disabled when you type in invalid amounts (too large or zero)
- you should get appropriate error messaging when you click into and out of the input without entering an amount

**Mainnet - prices for all tokens**
- repeat the above steps after checking out this branch locally and changing the contents of `.env` to `NETWORK="mainnet"`
- use caution when testing here as this is mainnet; only send coins to yourself if you dont want to lose currency

## Checklist:
